<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hyperlinkSecurityWarningDialog</class>
 <widget class="QDialog" name="hyperlinkSecurityWarningDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>69</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Security Warning</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0" rowspan="2">
    <widget class="QLabel" name="warningIconLabel">
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <spacer name="indentSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="messageLabel">
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <layout class="QHBoxLayout" name="buttonBoxLayout">
     <item>
      <widget class="QPushButton" name="learnMorePushButton">
       <property name="text">
        <string>Learn More</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttonBoxSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="doNotOpenPushButton">
       <property name="text">
        <string>Do Not Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openPushButton">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
